docs/contributing/how to contribute.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute
+
+We're grateful for your interest in participating in this library!
+
+## Asking questions
+
+If you have any question on this library, don't hesitate to ask them.
```

Comment:
```suggestion
If you have any question about this library, feel free to ask them.
```

Maybe you can suggest the Discussion tab in GitHub (if enabled) or the Slack channel ?

Reply:
Ah well, I should have read next.

Reply:
We've decided to not create a Discussion tab in GitHub just yet, as we don't want to fragment communications too much. If and when we get hundreds and thousands of users, we'll probably have to revise this strategy :)